about/introduction.rst

```diff
@@ -6,105 +6,98 @@ Introduction
+if you need a quick write-up about Godot Engine:
 
     Godot Engine is a feature-packed, cross-platform game engine to create 2D
     and 3D games from a unified interface. It provides a comprehensive set of
-    common tools, so users can focus on making games without having to
-    reinvent the wheel. Games can be exported in one click to a number of
-    platforms, including the major desktop platforms (Linux, macOS, Windows)
-    as well as mobile (Android, iOS) and web-based (HTML5) platforms.
-
-    Godot is completely free and open source under the permissive MIT
-    license. No strings attached, no royalties, nothing. Users' games are
-    theirs, down to the last line of engine code. Godot's development is fully
-    independent and community-driven, empowering users to help shape their
-    engine to match their expectations. It is supported by the `Godot
-    Foundation <https://godot.foundation/>`_ not-for-profit.
-
-For a more in-depth view of the engine, you are encouraged to read this
-documentation further, especially the :ref:`Getting Started <sec-learn>` series.
-
-About the documentation
------------------------
-
-This documentation is continuously written, corrected, edited, and revamped by
-members of the Godot Engine community. It is edited via text files in the
-`reStructuredText <https://www.sphinx-doc.org/en/stable/rest.html>`_ markup
-language and then compiled into a static website/offline document using the
-open source `Sphinx <https://www.sphinx-doc.org>`_ and `ReadTheDocs
-<https://readthedocs.org/>`_ tools.
-
-.. note:: You can contribute to Godot's documentation by opening issue tickets
-          or sending patches via pull requests on its GitHub
-          `source repository <https://github.com/godotengine/godot-docs>`_, or
-          translating it into your language on `Hosted Weblate
-          <https://hosted.weblate.org/projects/godot-engine/godot-docs/>`_.
-
-All the contents are under the permissive Creative Commons Attribution 3.0
-(`CC BY 3.0 <https://creativecommons.org/licenses/by/3.0/>`_) license, with
-attribution to "Juan Linietsky, Ariel Manzur and the Godot Engine community".
+    common tools, so that users can focus on making games without having to
+    reinvent the wheel. Games can be exported with one click to a number of
+    platforms, including the major desktop platforms (Linux, macOS, Windows),
+    mobile platforms (Android, iOS), as well as Web-based platforms and consoles.
+
+    Godot is completely free and open source under the :ref:`permissive MIT
+    license <doc_complying_with_licenses>`. No strings attached, no royalties,
+    nothing. Users' games are theirs, down to the last line of engine code.
+    Godot's development is fully independent and community-driven, empowering
+    users to help shape their engine to match their expectations.
+    It is supported by the `Godot Foundation <https://godot.foundation/>`_
+    not-for-profit.
+
```

Comment:
This was updated from SFC to Godot Foundation, and the copy-paste was also updated from the engine repo and now mentions consoles. Also added a link to the license page.